<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupSelection</class>
 <widget class="QWidget" name="GroupSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/game/jackpots/jackpots/woodBG2.jpg);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);</string>
     </property>
     <property name="text">
      <string>GROUPS MANAGER</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="groupsList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="groupNotifs">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
     <property name="text">
      <string>View Group Notifications</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/buttons/icons/icons/chat.png</normaloff>:/buttons/icons/icons/chat.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="joinGroup">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
     <property name="text">
      <string>Join a Group</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/buttons/icons/icons/join.png</normaloff>:/buttons/icons/icons/join.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="newGroup">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/buttons/icons/icons/new.png</normaloff>:/buttons/icons/icons/new.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="backB">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/buttons/icons/icons/back.png</normaloff>:/buttons/icons/icons/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="deleteGroup">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/buttons/icons/icons/delete.png</normaloff>:/buttons/icons/icons/delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="response">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
